app/assets/javascripts/discourse/components/d-editor.js.es6

```diff
@@ -434,6 +434,10 @@ export default Ember.Component.extend({
           const full = `:${term}`;
           term = term.toLowerCase();
 
+          if (term.length < Discourse.SiteSettings.emoji_trigger) {
```

Comment:
We try not to rely on globals anymore so we inject the site settings in most Ember object. You can access them directly with `this.siteSettings.emoji_trigger` ;)

Reply:
Thanks!